sdk/identity/identity/TROUBLESHOOTING.md

```diff
@@ -453,27 +457,39 @@ Once you've verified Azure PowerShell is using the correct account, validate tha
+
+## Troubleshoot `AzurePipelinesCredential` authentication issues
+
+| Error Message                                                                                                                                                                                                                                                                                                                                                            | Description                                                                                                                                                                       | Mitigation                                                                                                                                                                                                                                                                                                                                                                                                             |
+| --- | ---| --- |
+| AADSTS900023: Specified tenant identifier '<some tenant Id>' is neither a valid DNS name, nor a valid external domain.                                                                                                                                                                                                                                                   | The Microsoft Entra tenant ID passed to the credential is invalid.                                                                                                                | Verify the tenant ID is valid. If the service connection federated identity credential (FIC) was configured via a user-assigned managed identity, the tenant will be the one in which managed identity was registered. If the service connection FIC is configured via an app registration, the tenant should be the one in which the app registration is registered.                                                                                  |
```

Comment:
Let's backtick `<some tenant ID>` as right now it's being rendered as an HTML tag and doesn't appear.

```suggestion
| AADSTS900023: Specified tenant identifier `<some tenant ID>` is neither a valid DNS name, nor a valid external domain.                                                                                                                                                                                                                                                   | The Microsoft Entra tenant ID passed to the credential is invalid.                                                                                                                | Verify the tenant ID is valid. If the service connection federated identity credential (FIC) was configured via a user-assigned managed identity, the tenant will be the one in which managed identity was registered. If the service connection FIC is configured via an app registration, the tenant should be the one in which the app registration is registered.                                                                                  |
```